### PR TITLE
Fix PasswordField on postgres (fix #766)

### LINKED ISSUE #766: PasswordField does not seem to work with Postgres
```python
import peewee
from playhouse.fields import PasswordField

db = peewee.PostgresqlDatabase('peewee')


class User(peewee.Model):
    username = peewee.CharField(max_length=10)
    password = PasswordField()

    class Meta:
        database = db


if __name__ == '__main__':
    User.create_table()
    user = User.create(username='yo', password='password')
    user = User.get(User.id == user.id)
    assert user.password.check_password('password')
```
Fails with this traceback:
```
Traceback (most recent call last):
  File "peewee_password_field.py", line 22, in <module>
    user = User.get(User.id == user.id)
  File "/home/ybon/.virtualenvs/ban/lib/python3.4/site-packages/peewee.py", line 4393, in get
    return sq.get()
  File "/home/ybon/.virtualenvs/ban/lib/python3.4/site-packages/peewee.py", line 2812, in get
    return clone.execute().next()
  File "/home/ybon/.virtualenvs/ban/lib/python3.4/site-packages/peewee.py", line 2029, in next
    obj = self.iterate()
  File "/home/ybon/.virtualenvs/ban/lib/python3.4/site-packages/peewee.py", line 2015, in iterate
    return self.process_row(row)
  File "/home/ybon/.virtualenvs/ban/lib/python3.4/site-packages/peewee.py", line 2090, in process_row
    setattr(instance, column, func(row[i]))
  File "/home/ybon/.virtualenvs/ban/lib/python3.4/site-packages/playhouse/fields.py", line 66, in python_value
    return PasswordHash(value)
TypeError: string argument without an encoding

```

### PR DESCRIPTION
**Pull request by yohanboniface**
Fix #766

This also makes the PasswordField tests to be ran with postgres.

### CODE REVIEW

**Comment by yohanboniface** [comment]
@coleifer any feedback on this? :)

**Comment by foxx** [comment]
LGTM. I haven't tested with Postgres, but the changes looks sane. Changing from BaseModel to TestModel allows it to be ran with `--engine postgres`, and changing from TextField to BlobField allows bytes to be stored safely.

File: playhouse/tests/test_extra_fields.py [hunk 1]
```diff
@@ -129,7 +130,7 @@ def test_encrypted_field(self):
 @skip_if(lambda: PasswordField is None)
 class TestPasswordFields(ModelTestCase):
     def setUp(self):
-        class PasswordModel(BaseModel):
+        class PasswordModel(TestModel):
```
Comment (coleifer) [inline playhouse/tests/test_extra_fields.py]: This does not properly clean up after itself.

Comment (coleifer) [inline playhouse/tests/test_extra_fields.py]: Oh wait, yeah looks like it does.

**Comment by coleifer** [comment]
Thank you!

**Pull request merged**